### DISCUSSION TITLE
v7.3.0-rc.1 Release of PowerShell

### DISCUSSION THREAD

**Discussion by adityapatwardhan** [PowerShell/PowerShell#18396] [General]
### Breaking Change

- Update to use `ComputeCore.dll` for PowerShell Direct (#18194)

### Engine Updates and Fixes

- On Unix, explicitly terminate the native process during cleanup only if it's not running in background (#18215)

### General Cmdlet Updates and Fixes

- Remove the `ProcessorArchitecture` portion from the full name as it's obsolete (#18320)

### Tests

- Add missing `-Tag 'CI'` to describe blocks. (#18317)

### Build and Packaging Improvements

<details>

<summary>
<p>Bump to .NET 7 to <code>7.0.100-rc.2.22477.20</code> (#18328)(#18286)</p>
</summary>

<ul>
<li>Update ThirdPartyNotices (Internal 22987)</li>
<li>Remove API sets (#18304) (#18376)</li>
<li>Do not cleanup <code>pwsh.deps.json</code> for framework dependent packages (#18300)</li>
<li>Bump <code>Microsoft.PowerShell.Native</code> from <code>7.3.0-preview.1</code> to <code>7.3.0-rc.1</code> (#18217)</li>
<li>Remove unnecessary native dependencies from the package (#18213)</li>
<li>Make the link to minimal package blob public during release (#18158)</li>
<li>Create tasks to collect and publish hashes for build files. (#18276)(#18277)</li>
<li>Add branch counter to compliance build (#18214)</li>
<li>Move APIScan to compliance build (#18191)</li>
<li>Update MSI exit message (#18137)</li>
<li>Remove XML files for min-size package (#18189)</li>
</ul>

</details>

[7.3.0-rc.1]: https://github.com/PowerShell/PowerShell/compare/v7.3.0-preview.8...v7.3.0-rc.1

### SHA256 Hashes of the release artifacts

- hashes.sha256
  - FA3A1086B5742D63C84E62ED7D6DC61D7D0AF6771341AC59A3FB473061186173
- powershell-7.3.0-rc.1-linux-alpine-x64.tar.gz
  - C800018ED3DC809023503775C7E4E66B659E45EC37DC769646CA06DE9D7E8D31
- powershell-7.3.0-rc.1-linux-arm32.tar.gz
  - BAB2A23E68C84075D9B492B35558AF16ECB85DE8B2FC0B5B5AFC8CE452C56A3B
- powershell-7.3.0-rc.1-linux-arm64.tar.gz
  - BE02398DE3457F9A61942291B1CDA5030A6CD5F4ADE0F96F20797C614E1F3844
- powershell-7.3.0-rc.1-linux-x64.tar.gz
  - 0FF8D846E071A3D7C04D854BBC4582F9928D56500B0976504A520DACDECFBC6F
- powershell-7.3.0-rc.1-linux-x64-fxdependent.tar.gz
  - F53E90ED3E6C74239E16EBCDD232F2EE6AAD34E577B281959F347ECCB6001151
- powershell-7.3.0-rc.1-osx-arm64.pkg
  - DF946C85EFD17D3712C281D722CFCC4177EE7CBFE5F462D664521A567D81CF0F
- powershell-7.3.0-rc.1-osx-arm64.tar.gz
  - CE116DC4E4132EF6534DF3E3FD68B9189AE55E67BD171A2160D3BB94A1D9349F
- powershell-7.3.0-rc.1-osx-x64.pkg
  - 4C1C5658C86952C239D330AFABAD6502531996AE213AB5DEA74EFD767122CB82
- powershell-7.3.0-rc.1-osx-x64.tar.gz
  - DD1405D81B2AAB5AEE1CC62540D898A666D975235DB2DE7DBD0E2F88826CE079
- PowerShell-7.3.0-rc.1-win-arm32.zip
  - F0F8C62DCD9FC8073765ED4E24D2ADACE4608B925D5E6D3DF2C01C8B5BECBDA0
- PowerShell-7.3.0-rc.1-win-arm64.zip
  - 5DECE383005E94E60ED0A63066FBF6D998F43E4063EC2697C9B17B17A345F596
- PowerShell-7.3.0-rc.1-win-fxdependent.zip
  - 3EF0F5051F0A4543CCEDB412D4034CE6FC3EF2F0269F404FFFCDB243E2BD236C
- PowerShell-7.3.0-rc.1-win-fxdependentWinDesktop.zip
  - 1F4C7A46BA9D882F23FFD289F03B9C92994097A1DA7D1F19B50B066BB4C773C5
- PowerShell-7.3.0-rc.1-win-x64.msi
  - 2C304FEDCD04BD1F9040BA6DD0B6AEADA9248C7BB9B344103E1B29440793766F
- PowerShell-7.3.0-rc.1-win-x64.zip
  - CD0AF8921B19FA93A894ACA9AFB5155417D2B23C8D9A573B9216B607D95E9EC7
- PowerShell-7.3.0-rc.1-win-x86.msi
  - F23DF935F22513443606B261E0EE696F3AC22A6CE77F1FF6EDE788F67E08BC60
- PowerShell-7.3.0-rc.1-win-x86.zip
  - 4BC9322312885229AFE6BE8E08EB91832A7035BFCA78B58F02B60F4343F463C1
- powershell-preview_7.3.0-rc.1-1.deb_amd64.deb
  - 5CCA8118D8E2238BD4C7ED40A2CD370C12320EA5096AA5DFC6AB4911F3D6E7F1
- powershell-preview-7.3.0_rc.1-1.cm.x86_64.rpm
  - B63701909FFE6847E84954B42BD6F2860E690F2DE90408AB8E7EED456311950E
- powershell-preview-7.3.0_rc.1-1.rh.x86_64.rpm
  - 621FD3A034AC637E66ECB0113B542B7DDFE2DB0A66FABC04614DAAD8A074773B

<hr /><em>This discussion was created from the release <a href='https://github.com/PowerShell/PowerShell/releases/tag/v7.3.0-rc.1'>v7.3.0-rc.1 Release of PowerShell</a>.</em>

**Comment by warshanks** [comment]
Any chance of a .msixbundle for arm/arm64?

**Comment by haritha99ch** [comment]
Both **MS Store** and **msixbundle** installations **fail** with **0x87E10BC6**. Can someone look into this?
